sdk/tables/azure-data-tables/azure/data/tables/_entity.py

```diff
@@ -75,17 +76,40 @@ class EntityProperty(object):
     """
 
     def __init__(self,
+                value=None, # type: any
```

Comment:
tiniest nit: the indentation here is a little weird, it should be two tabs to the left